### ISSUE TITLE
I want Pull Requests to be squashed when they get merged

### ISSUE THREAD

**Issue by cvchaparro** [jondavid-black/AaC#229]
As an AaC repository maintainer, I want Pull Requests to be squashed when they get merged so that the repository is always in a working state and the commit history is clean.

## AC

- [ ] When I complete a Pull Request, it is accepted, and I click the merge button in the GitHub interface, the commits are squashed and the merge is performed.

## Resources

- [Configuring commit squashing for pull requests](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests)

**Comment by Coffee2Bits** [comment]
Good call! I was just thinking about doing this after playing with the release process.

**Comment by Coffee2Bits** [comment]
I think the only person who currently has the necessary permissions to complete this is @jondavid-black.
